containers/ws/release.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
No need to change to bash, all looks POSIXy here. But not important.

Reply:
Ah yes, that was only necessary in one of my QEMU-related drafts for iterating over comma-separated architectures.